File: fretecalc/app/src/main/java/com/example/fretecalc/models/prices/PriceResponse.java

package com.example.fretecalc.models.prices;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.Expose;

@Entity(tableName = "prices")
public class PriceResponse {

    @PrimaryKey
    private Long priceId;
    @Expose
    private Double frigorificada;
    @Expose
    private Double geral;
    @Expose
    private Double granel;
    @Expose
    private Double neogranel;
    @Expose
    private Double perigosa;

    public PriceResponse() {
    }

    public Long getPriceId() {
        return priceId;
    }

    public void setPriceId(Long priceId) {
        this.priceId = priceId;
    }

    public Double getFrigorificada() {
        return frigorificada;
    }

    public void setFrigorificada(Double frigorificada) {
        this.frigorificada = frigorificada;
    }

    public Double getGeral() {
        return geral;
    }

    public void setGeral(Double geral) {
        this.geral = geral;
    }

    public Double getGranel() {
        return granel;
    }

    public void setGranel(Double granel) {
        this.granel = granel;
    }

    public Double getNeogranel() {
        return neogranel;
    }

    public void setNeogranel(Double neogranel) {
        this.neogranel = neogranel;
    }

    public Double getPerigosa() {
        return perigosa;
    }

    public void setPerigosa(Double perigosa) {
        this.perigosa = perigosa;
    }

}

File: app/src/main/java/e/caioluis/android_case/json/PriceResponse.kt
package e.caioluis.android_case.json

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class PriceResponse(

    @SerializedName("frigorificada")
    val frigorificada: Double,
    @SerializedName("geral")
    val geral: Double,
    @SerializedName("granel")
    val granel: Double,
    @SerializedName("neogranel")
    val neogranel: Double,
    @SerializedName("perigosa")
    val perigosa: Double

) : Serializable